lib/client/js/ui/BeansView.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * BeansView class that extends from the base Control class to
+ * dynamically build and update a UI container that will
+ * house the main menu and content sections.
+ *
+ * @class
+ */
+var BeansView = (function () {
+    /**
+     * @alias BeansView
+     */
+    function BeansView() {
+        /** @member {Object} */
+        /** @member {Object} */
```

Comment:
These doclets should come directly above the variables themselves. Also, since these are instances, we can put their actual types instead of just `Object` - `@member {Menu}`.